### PR TITLE
sday20.py

### PR DESCRIPTION
**Pull request by saivarunkolluru**
DAy 20 strings

### CODE REVIEW

File: PYTHON2/day20/sday20.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import re
+str=raw_input()
```
Comment (gshguru) [inline PYTHON2/day20/sday20.py:2]: `str` is reserved symbol in python

File: PYTHON2/day20/sday20.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+str=str.replace('[',' ')
+str=str.replace(']',' ')
+str=str.replace('\\',' ')
+#print str
+s=re.split("!|[|,|?|.|_|'|@|+|]| ",str.strip())
```
Comment (gshguru) [inline PYTHON2/day20/sday20.py:7]: If you have already replaced `[` and `]` with spaces, why do we need again in regex too?

File: PYTHON2/day20/sday20.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+print len(s)-s.count('')-s.count(' ')
```
Comment (gshguru) [inline PYTHON2/day20/sday20.py:8]: There is a util  `filter(None, myList)` in python. Have a look

File: PYTHON2/day20/sday20.py (hunk 2 above)
Comment (saivarunkolluru) [inline PYTHON2/day20/sday20.py]: they are no included in the regex

File: PYTHON2/day20/sday20.py (hunk 3 above)
Comment (saivarunkolluru) [inline PYTHON2/day20/sday20.py]: thank u

File: PYTHON2/day20/sday20.py (hunk 1 above)
Comment (saivarunkolluru) [inline PYTHON2/day20/sday20.py]: nothing happens

Comment (saivarunkolluru) [inline PYTHON2/day20/sday20.py:2]: nothing happens unless i import str

File: PYTHON2/day20/sday20.py (hunk 3 above)
Comment (saivarunkolluru) [inline PYTHON2/day20/sday20.py]: using filter may lead to problem sometimes

File: PYTHON2/day20/sday20.py (hunk 1 above)
Comment (della-90) [inline PYTHON2/day20/sday20.py:2]: That's not true: you are forcing `str` (which is a type) to be an actual string. Try to call for example `print str(2)` to see what happens

File: PYTHON2/day20/sday20.py (hunk 2 above)
Comment (gshguru) [inline PYTHON2/day20/sday20.py:7]: What is the meaning of `|[|` and `|]|` in 7th line?

File: PYTHON2/day20/sday20.py (hunk 3 above)
Comment (gshguru) [inline PYTHON2/day20/sday20.py:8]: Do you have any example where filter is problematic?

File: PYTHON2/day20/sday20.py (hunk 1 above)
Comment (gshguru) [inline PYTHON2/day20/sday20.py:2]:
``` python
Python 2.7.10 (default, Jul 14 2015, 19:46:27) 
[GCC 4.2.1 Compatible Apple LLVM 6.0 (clang-600.0.39)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import str
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: No module named str
```

**Pull request closed without merging**